src/StoreClient.h

```diff
@@ -67,17 +68,23 @@ class store_client
 public:
     store_client(StoreEntry *);
     ~store_client();
-    bool memReaderHasLowerOffset(int64_t) const;
+    /// \returns the smallest of the passed value and the memory offset of the client
+    int memHeaderOffsetLowerThan(const int64_t offset) const;
```

Comment:
AFAICT, this method looks at "the memory offset of the client" only for `STORE_MEM_CLIENT` clients. The description does not seem to reflect that strange fact. However, see the other change request for this method first.